server/graphql/types/src/types/asset_catalogue_item.rs

```diff
@@ -0,0 +1,84 @@
+use async_graphql::*;
+
+use graphql_core::simple_generic_errors::NodeError;
+use repository::assets::{
+    asset_catalogue_item::AssetCatalogueItem, asset_catalogue_item_row::AssetCatalogueItemRow,
+};
+use service::ListResult;
+
+#[derive(PartialEq, Debug)]
+pub struct AssetCatalogueItemNode {
+    pub asset_catalogue_item: AssetCatalogueItem,
+}
+
+#[derive(SimpleObject)]
+pub struct AssetCatalogueItemConnector {
+    total_count: u32,
+    nodes: Vec<AssetCatalogueItemNode>,
+}
+
+#[Object]
+impl AssetCatalogueItemNode {
+    pub async fn id(&self) -> &str {
+        &self.row().id
+    }
+
+    pub async fn asset_category_id(&self) -> &str {
```

Comment:
We might want loaders for `asset_category` etc...
@mark-prins do you need this? Good idea to do it in another PR/Issue though.